packages/astro/test/fixtures/reroute/src/pages/reroute.astro

```diff
@@ -0,0 +1,11 @@
+---
+return Astro.rewrite("/")
```

Comment:
Is it worth testing rewrites to reserved urls like `/404` and `/500` as well? Only ask since Astro special cases these urls.

Reply:
Commit [`78b13f8` (#10867)](https://github.com/withastro/astro/pull/10867/commits/78b13f85da870b280c273220ae722e4bd78aa30f) adds tests for 404 rewrite